client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/IssueService.java

```diff
@@ -82,4 +82,11 @@ public interface IssueService {
    */
   @JsonRequest
   CompletableFuture<CheckStatusChangePermittedResponse> checkStatusChangePermitted(CheckStatusChangePermittedParams params);
+
+
+  @JsonRequest
+  CompletableFuture<Boolean> reopenIssue(ReopenIssueParams params);
+
+  @JsonRequest
+  CompletableFuture<Boolean> reopenAllIssuesForFile(ReopenAllIssuesForFileParams params);
```

Comment:
Lately we are trying to document methods in service with JavaDoc, you can have a look at other methods for inspiration